ArduCopter/ekf_check.cpp

```diff
@@ -35,7 +35,7 @@ void Copter::ekf_check()
     }
 
     // return immediately if motors are not armed, ekf check is disabled, not using ekf or usb is connected
-    if (!motors->armed() || ap.usb_connected || (g.fs_ekf_thresh <= 0.0f)) {
+    if (!motors->armed() || (g.fs_ekf_thresh <= 0.0f)) {
```

Comment:
Comment above should be updated since the usb connected check is being removed.

Reply:
Fixed.  It was already stuffed - I just stuffed it further...